### PR TITLE
Add node 6 in package.json, rename package and update gitignore

### PR DESCRIPTION
**Pull request by guillett**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,3 +2,4 @@ language: node_js
 node_js:
     - "4.2"
     - "5"
```
Comment (MattiSG) [inline .travis.yml]: Let's delete 5, that was an unstable release.

File: package.json [hunk 2]
```diff
@@ -42,7 +43,7 @@
     "webpack": "^1.12.14"
   },
   "engines": {
-    "node": "^4.2.0"
+    "node": "^4.2.0 || ^6.9.0"
```
Comment (MattiSG) [inline package.json]: Huh? We're testing for Node 5. Why isn't this `>= 4.2.0`?

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sgmap.ludwig",
+  "name": "sgmap-ludwig",
```
Comment (MattiSG) [inline package.json]: A justification for the rename would be more than welcome. This impacts all reusers (even if there are none today, a note in the commit message seems mandatory to me).

If we rename, let's use [scoped packages](https://docs.npmjs.com/misc/scope) and publish this as `@sgmap/ludwig`.

Comment (guillett) [inline package.json]: I had a lot of trouble to use that package and I was worried that the name was part of the issue. I also thought that the `.` was not standard (cf Mes Aides packages).

Your suggestion is much better, I will move toward it.

**Comment by MattiSG** [comment]
> fixup! 7e2acaf

I recommend to use commit messages rather than hashes.
First of all, it makes it more readable.
But most importantly, if you do a partial rebase, insert a commit, basically change the history in any way, you're changing the SHA.

**Pull request merged**